README.md

```diff
@@ -331,3 +331,29 @@ relatively long time to execute them.  Here is how you invoke the tests:
 If the tests pass then the execution will succeed and there will be no output.
 These tests may take some time to execute on Thor.  They run much faster on
 either hthor or ROXIE, due to the use of small inline datasets.
+
+### Report
+
+A report is generated based on the output of `Profile()`. The report is accessible via a Workunit's *Resources* tab within ECL Watch. For example:
+https://play.hpccsystems.com:18010/?Widget=WUDetailsWidget&Wuid=W20190430-175856#/stub/Resources-DL/Grid
```

Comment:
Add comment explaining this link was valid at time of writing and mark that WU as protected in ECL Watch

Reply:
Actually, I would suggest removing this link and replacing it with a screenshot showing a workunit's tabs, with "Resources" active and the tab label itself highlighted/circled.